conda/meta.yaml

```diff
@@ -1,13 +1,33 @@
 {% set data= load_setup_py_data() %}
 package: 
-  name: "ts-integrationtests"
+  name: ts-integrationtests
   version: {{ data.get('version') }}
 
 source:
   path: ..
 
 build:
-  script: python -m pip install --no-deps --ignore-installed .
+  noarch: generic
+  script: python -m pip install --ignore-installed .
+  entry_points:
+    - auxtel_disabled_enabled = lsst.ts.IntegrationTests.auxtel_disabled_enabled:run_auxtel_disabled_enabled
+    - auxtel_offline_standby = lsst.ts.IntegrationTests.auxtel_offline_standby:run_auxtel_offline_standby
+    - auxtel_prepare_for_flatfield = lsst.ts.IntegrationTests.auxtel_prepare_for_flatfield:run_auxtel_prepare_for_flatfield
+    - auxtel_prepare_for_onsky = lsst.ts.IntegrationTests.auxtel_prepare_for_onsky.:un_auxtel_prepare_for_onsky
```

Comment:
This doesn't look right: ...`onsky.:un_auxtel`...

Reply:
Fixed.